scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -138,8 +138,9 @@ SG_ DUMMY : 0|1@1+ (1,0) [0|1] "" DEBUG
 BO_ 311 FSM_STEERING_ANGLE_SENSOR: 4 FSM
 SG_ Steering_Angle : 0|32@1+ (1,0) [-110|110] "deg" DCM
 
-BO_ 312 FSM_BRAKE_PRESSURE_SENSOR: 4 FSM
+BO_ 312 FSM_BRAKE_PRESSURE_SENSOR: 5 FSM
```

Comment:
I think with the `App_Brake`, we can just combine CAN message 308 and 312 to just `BO_ 308 FSM_BRAKE:`

i.e.
```
BO_ 308 FSM_BRAKE: 8 FSM
SG_ Is_Brake_Actuated ...
SG_ Is_Pressure_Sensor_Open_Or_Short_Circuit ...
SG_ Brake_Pressure ...
```

Reply:
ok this is done